BedrockServer.cpp

```diff
@@ -920,6 +936,11 @@ void BedrockServer::postPoll(fd_map& fdm, uint64_t& nextActivity) {
             break;
             case STCPManager::Socket::CONNECTED:
             {
+                // We try and read here, so that we're sure to have all the latest data from newly connected sockets.
+                // This is primarily a concern when we're trying to shut down, but had still receiving new requests
+                // right up until the end.
+                s->recv();
```

Comment:
This will probably have a performance impact because it's reading from every socket every loop -- even if `select()` doesn't indicate there is anything to read.  I don't think we need to do this (do we have reason to believe sometimes there's data even if `select()` claims there isn't?), and in fact I think doing this will be unacceptably slow due to `recv()` being a relatively slow kernel function in a tight loop